src/index.html

```diff
@@ -6,7 +6,362 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <!-- #region header -->
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo__image"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
```

Comment:
Let's add corresponding hrefs according to the content
```suggestion
              href="#samsung"
```